<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgBcode</class>
 <widget class="QDialog" name="dlgBcode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRIG-B码(DC)对时</string>
  </property>
  <widget class="QLineEdit" name="lineEditBcodeTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSettime">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>手动校时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkEnAutoSet">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>使能自动校时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自动校时周期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IRIG-B码时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前系统时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCurTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInterval">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenSerial">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>开始接收</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
